docs/src/quick_start.md

```diff
@@ -0,0 +1,66 @@
+# Quick start
+
+This code:
+
+```jldoctest cpu
```

Comment:
why are these doctests? was this on purpose?

Reply:
hmm no. I think I want to literate this thing.

Reply:
```@example

?